packages/stacked_tools/lib/src/models/config_model.dart

```diff
@@ -24,7 +24,7 @@ class Config with _$Config {
     /// Relative path for test helper file that contains the register functions
     /// for unit test setup and mock declarations.
     @JsonKey(name: 'test_helpers_path')
-    @Default('test/helpers/test_helpers.dart')
+    @Default('test/helpers')
```

Comment:
Adding a new test file does some weird stuff with the imports. I wonder if this will fix it.

Reply:
I think with this changes would work fine.